README.md

```diff
@@ -1150,13 +1150,13 @@ Capturing group must be an indication that it's used somewhere in the code. Any
 
 Don't capture id attribute name here since you can't use it for anything anyway.
 
-Correct:
+###### Correct:
```

Comment:
This will result in different `<hX>`'s for `Correct:` and `Incorrect:` across the document.

Reply:
If I picked the deepest value for all of them, would that be ok? (initially, I'm going to kick these hunks into their own commit and drop them from this PR)